scripts/setup.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/sh -ex
+export PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+export DEBIAN_FRONTEND=noninteractive
+export ETCKEEPER_NO_PUSH=1
+
+die() {
+    echo "$@" >&2
```

Comment:
I'm kinda confused: we already display an error message and return a non-zero exit value.
What more can we possible add there?